bigcache.go

```diff
@@ -58,6 +59,14 @@ func newBigCache(config Config, clock clock) (*BigCache, error) {
 		cache.shards[i] = initNewShard(config, onRemove)
 	}
 
+	if config.CleanWindow >= time.Second {
```

Comment:
Our timestamp is in unix epoch. Any tick <1 second is sort of wasting CPU.

Anyway, it's not intuitive. I'll change it.